app.rb

```diff
@@ -1,8 +1,32 @@
+# CONTROLLER
+# Get data from the model and put it in the view
```

Comment:
the manner you have written the code along with comments and user story has made it much easier and understandable for me